cardano-config/src/Cardano/Config/Cardano/Protocol.hs

```diff
@@ -178,7 +178,10 @@ mkConsensusProtocolCardano NodeByronProtocolConfiguration {
         shelleyLeaderCredentials
 
         -- Hard fork parameters
-        Consensus.NoHardCodedTransition --TODO
+        (Just 190) --TODO: Optimisation: once the epoch of the transition is
+                   -- known, set this to the first shelley epoch.
+        (Consensus.NoHardCodedTransition (fromIntegral npcShelleyMaxSupportedProtocolVersion))
+        --TODO is this the right value?
```

Comment:
Should we remove this particular `TODO`?

```suggestion
```

Reply:
Actually, it seems that a `TODO` was there beforehand so nevermind.

Reply:
Yeah, I'm actually still not sure about that one. So a valid remaining TODO. But we'll get to testing this soonish, but not this PR.

Reply:
The argument to `NoHardCodedTransition` should be the major version number we will use for Shelley, I believe that is 2.